### PR TITLE
Atkinson dithering

### PR DESCRIPTION
**Pull request by litlnemo**
As a Mac user in the 80s I wanted to add Atkinson dithering for old times' sake. It's not _that_ different from Floyd Steinberg, but there is a difference. 

Also changed a couple of typos in the comments that I noticed.

### CODE REVIEW

**Comment by litlnemo** [comment]
Oh, by the way, I tested on an MX11 cat printer. No problems other than that the prints are lighter and less contrasty than the ones from Fun Print.

File: catprinter/img.py [hunk 1]
```diff
@@ -23,15 +23,42 @@ def adjust_pixel(y, x, delta):
             new_val = 255 if img[y][x] > 127 else 0
             err = img[y][x] - new_val
             img[y][x] = new_val
-            adjust_pixel(y, x + 1, err * 7/16)
+            adjust_pixel(y, x + 1, err * 1/16)
```
Comment (rbaron) [inline catprinter/img.py]: I'm not 100% sure, but I think this was supposed to really be `7` (at least according to [wiki](https://en.wikipedia.org/wiki/Floyd%e2%80%93Steinberg_dithering)).

**Comment by rbaron** [comment]
Thanks @litlnemo. I just gave `-b atkinson` a go and it looks great.

File: catprinter/img.py (hunk 1 above)
Comment (litlnemo) [reply catprinter/img.py]: Oh, that might be a copy/paste error on my part. Good eye! I didn't intend to change the Steinberg dithering at all. Thanks!

**Review by rbaron — APPROVED** [review]
Thanks!

**Pull request merged**